### PR TITLE
Our of Memory fix when streaming large files

### PR DESCRIPTION
**Pull request by BaronGreenback**
This is a fix for an out of memory error with large files. The solution implmented is how the other streams perform the action.

### CODE REVIEW

**Comment by crobibero** [comment]
Does this fix the out or memory error, or add it?

**Comment by BaronGreenback** [comment]
New code fixes it.

File: Jellyfin.Api/Helpers/FileStreamResponseHelpers.cs [hunk 1]
```diff
@@ -123,10 +123,14 @@ public static class FileStreamResponseHelpers
...
-                memoryStream.Position = 0;
-                return new FileStreamResult(memoryStream, contentType);
+                await new ProgressiveFileCopier(outputPath, job, transcodingJobHelper, CancellationToken.None)
+                    .WriteToAsync(httpContext.Response.Body, CancellationToken.None).ConfigureAwait(false);
+                return new FileStreamResult(httpContext.Response.Body, contentType);
+
+                // var memoryStream = new MemoryStream();
+                // await new ProgressiveFileCopier(outputPath, job, transcodingJobHelper, CancellationToken.None).WriteToAsync(memoryStream, CancellationToken.None).ConfigureAwait(false);
+                // memoryStream.Position = 0;
+                // return new FileStreamResult(memoryStream, contentType);
```
Comment (crobibero) [inline Jellyfin.Api/Helpers/FileStreamResponseHelpers.cs:133]:
```suggestion
```

**Comment by crobibero** [comment]
Looks like another PR's changes snuck in as well

File: Jellyfin.Api/Helpers/FileStreamResponseHelpers.cs (hunk 1 above)
Comment (BaronGreenback) [reply Jellyfin.Api/Helpers/FileStreamResponseHelpers.cs:133]: Bad habbit of mine - when testing - in case anything goes wrong

**Review by crobibero — APPROVED** [review]

**Review by cvium — APPROVED** [review]

**Pull request merged**